mythril/laser/ethereum/instructions.py

```diff
@@ -941,7 +944,11 @@ def sha3_(self, global_state: GlobalState) -> List[GlobalState]:
         if data.symbolic:
             argument_str = str(state.memory[index]).replace(" ", "_")
             result = symbol_factory.BitVecFuncSym(
-                "KECCAC[{}]".format(argument_str), "keccak256", 256, input_=data
+                "KECCAC[{}]".format(argument_str),
+                "keccak256",
+                256,
+                input_=data,
+                annotations=state.memory[index].annotations,
```

Comment:
Shouldn't all the annotations in the range from memory[index:index+length] be combined in here? or I am missing something?